src/app/component/projects/projects.menus.ts

```diff
@@ -72,7 +72,7 @@ export const editProjectMenuItem = MenuRoute({
 });
 
 export const exploreAudioProjectMenuItem = MenuLink({
-  uri: "REPLACE_ME",
+  uri: params => `/visualize?projectId=${params.projectId}`,
```

Comment:
use our string templating function that was made for api services... 

Many different pages may want to create a visualize link.